### PR TITLE
Add portfolio : Mobile first

### PR DESCRIPTION
**Pull request by arnoldnekemiah**
Mobile first portfolio with Linters configured

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        <nav class="dropdown">
+          <img src="./Images/Icon - Menu.png" alt="">
+        </nav>
+      </header>
+      <section class="headline">
+        <div class="headline-container">
+          <h1>Hey There. I'm Arnold</h1>
+          <p>
+            <span>I’m a Software Developer</span>
+          </p>
```
Comment (TadesseDev) [inline index.html]: - [ ] I'm impressed with your design @arnoldnekemiah, It's also great you used an `h1` tag for the main title. But for the subtitle ("I’am a Software Developer"), we should use some heading tag other than `h1`. that can be from `h2` to `h6`.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+/* headline section */
+
+.headline {
+  background: #1c1a19;
+  color: #fff;
+  padding: 130px 25px 145px;
+  background-image: url(./Images/Group.png);
+  background-position: bottom right;
+  background-repeat: no-repeat;
+}
```
Comment (TadesseDev) [inline style.css]: - [ ] Dear @arnoldnekemiah, It's among the primary requirement to use flex-box to align contents inside this section. This gives us a lot of options to align all our elements, plus, it's responsive.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @arnoldnekemiah 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7


## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] Please try to capitalize your commits and be more specific in what you did. 
- [Optional] For your pull request, try to discuss the changes in more detail. This gives information about what changes and what has been updated to whoever is looking at your pull request. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arnoldnekemiah** [comment]
I have resolved the  issues as suggested in the comments

**Review by ShadyShawkat — APPROVED** [review]
## Hi @arnoldnekemiah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Good overall design ✔️

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**